design_docs/README.md

```diff
@@ -0,0 +1 @@
+Placeholder for actual design doc process
```

Comment:
Should we put this into `doc` instead of `design_doc`.

Reply:
I thought about that, but then doc is git ignored for some reason, so I just made a new one. I can ask Joy about this.